### PR TITLE
[fix](load) skip dcheck num short key columns if cluster key index is empty

### PR DESCRIPTION
**Pull request by kaijchen**
## Proposed changes

Skip dcheck num short key columns if cluster key index is empty.

Print more messages if DCHECK fails.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by kaijchen** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

File: regression-test/pipeline/cloud_p0/conf/regression-conf-custom.groovy [hunk 1]
```diff
@@ -16,7 +16,6 @@
 // under the License.
 
 testGroups = "p0"
```
Comment (kaijchen) [inline regression-test/pipeline/cloud_p0/conf/regression-conf-custom.groovy:18]:
```suggestion
testGroups = "p0"

```

**Comment by kaijchen** [comment]
run buildall

**Pull request closed without merging**